test/test_site/expected/bugs/index.html

```diff
@@ -12,8 +12,9 @@
     <link rel="stylesheet" href="..\markbind\css\bootstrap-glyphicons.min.css" >
     <link rel="stylesheet" href="..\markbind\css\github.min.css">
     <link rel="stylesheet" href="..\markbind\css\markbind.css">
+    <link rel="stylesheet" href="_markbind\layouts\default\styles.css">
```

Comment:
I think it would be better to put these inside the `markbind` folder (no underscroll at the front) rather than inside the `_markbind` folder, because the `_markbind` folder has a connotation that it contains working files, rather than files for actual deployment.

Same thing for the JavaScript files.

Reply:
Sure, moved it to the assets folder.

Also added path to ignore so they don't get copied to the _markbind folder